apps/nextjs-website/src/lib/api.ts

```diff
@@ -47,12 +77,25 @@ export async function getGuide(
   };
 }
 
+function getMainGuide(path?: string) {
+  const guidePath = path?.split('/').filter((p, index) => index < 4);
```

Comment:
What does the 4 number represent? I think you should replace it with a constant or at least write a little comment